### PR TITLE
Various fixes for ApacheConnector

### PR DESCRIPTION
**Pull request by felfert**
- Fix typo in CMakeLists.txt
 - Adapt ApacheConnector to apache httpd 2.4
 - Fix target naming (no lib prefix) and install target
 - Fix build on windows
 - Use recommended .so suffix  for mod_poco even on Windows
 - Handle quoted dll name in AddPocoRequestHandler command
 - Handle quoted string in AddPocoConfig command
 - Disable chunked transfer encoding in samples and doc

Fixes (all except chunked transfer encoding) #3079 

Note:
  Build tested on windows against the [ApacheLounge](https://www.apachelounge.com/download/VC15/) binary apache distribution (the one from ApacheHaus unfortunately is missing some headers) using the following options:

````
-DAPACHE2_ROOT_DIR=/Apache24 -DAPR_ROOT_DIR=/Apache24 -DAPRUTIL_ROOT_DIR=/Apache24 -DENABLE_APACHECONNECTOR=ON
````
Cheers
 -Fritz

### CODE REVIEW

File: ApacheConnector/CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +21,14 @@ target_include_directories(mod_poco
 		${APACHE2_INCLUDE_DIRS}
 		${CMAKE_CURRENT_SOURCE_DIR}/src
 )
-target_link_libraries(mod_poco PUBLIC Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
+set(DEPLIBS Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
+if(WIN32)
+	list(APPEND DEPLIBS Apache::httpd)
+endif(WIN32)
+target_link_libraries(mod_poco PUBLIC ${DEPLIBS})
```
Comment (Bjoe) [inline ApacheConnector/CMakeLists.txt:28]:
```suggestion
target_link_libraries(mod_poco PUBLIC Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
if(WIN32)
	target_link_libraries(mod_poco PUBLIC Apache::httpd)
endif(WIN32)
```
Keep it simple :-)

File: ApacheConnector/CMakeLists.txt [hunk 2]
```diff
@@ -19,7 +21,14 @@ target_include_directories(mod_poco
...
+
+file(MAKE_DIRECTORY "include/Poco")
```
Comment (Bjoe) [inline ApacheConnector/CMakeLists.txt:30]: Not sure why we need this ? Should be created with this `POCO_INSTALL` macro

**Review by Bjoe — CHANGES_REQUESTED** [review]
Please fix findings

File: ApacheConnector/CMakeLists.txt (hunk 1 above)
Comment (felfert) [reply ApacheConnector/CMakeLists.txt:28]: I think this suggestion is wrong.
AFAIK, target_link_libraries() is **NOT** cumulative. So: your suggestion would result in missing link libraries
Poco::Util Poco::Net,  Apache::Apr and Apache::Aprutil
if building on Windows?!

File: ApacheConnector/CMakeLists.txt (hunk 2 above)
Comment (aleks-f) [reply ApacheConnector/CMakeLists.txt:30]: @felfert what is the purpose of this line?

**Pull request closed without merging**